app/jinja_filters.py

```diff
@@ -695,6 +693,14 @@ def map_list_collector_config(
                 }
             )
 
+        icon = (
```

Comment:
Were we supposed to remove the person icon from list collector? Primary person and old lc will be affected, I can't find any mention on your card.

Reply:
Yeah fair point other List Collector without repeating Blocks will lose the Person icon. Will modify to default to Person if not repeating blocks, the check or None if complete or not with repeating blocks.

Reply:
Actually I think I did go over this with Mebin, we were to remove the icons because the person is appropriate for all lists. I'll document this in the trello card, but I'm sure that was the decision, my bad for not documenting on the card. There is mention of it on my [mini PR](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1112) which was approved by Mebin.

Reply:
Worth confirming with Mebin.